src/interfaces/records/handlers/records-delete.ts

```diff
@@ -12,7 +13,7 @@ import { RecordsWrite } from '../messages/records-write.js';
 
 export class RecordsDeleteHandler implements MethodHandler {
 
-  constructor(private didResolver: DidResolver, private messageStore: MessageStore, private dataStore: DataStore) { }
+  constructor(private didResolver: DidResolver, private messageStore: MessageStore, private dataStore: DataStore, private eventLog: EventLog) { }
```

Comment:
Sanity: `eventLog.append` is not invoked for delete?

Reply:
good catch